lib/types/platform/gitlab/index.ts

```diff
@@ -12,3 +13,10 @@ export type GitlabTreeNode = {
   id: string;
   name: string;
 } & GitTreeNode;
+
+export const GITLAB_API_USING_HOST_TYPES = [
+  PLATFORM_TYPE_GITLAB,
+  'gitlab-releases',
+  'gitlab-tags',
+  'pod',
```

Comment:
Won't this now "hijack" all pod to use gitlab auth?

Reply:
```suggestion
```

Yes i think so, not sure why it's added. So suggest to remove that

Reply:
Ah sry, I simply adapted the Github plattform example and haven't realized that `pod` is referencing a datasource. My Impressions has been that that has something to do that the Renovate App is probably deployed inside of a K8 cluster.